cmd/server-main.go

```diff
@@ -403,14 +414,7 @@ func serverMain(c *cli.Context) {
 	fatalIf(err, "Unable to parse storage endpoints %s", c.Args())
 
 	// Should exit gracefully if none of the endpoints passed as command line argument is local to this server.
-	anyLocalEp := false
-	for _, ep := range endpoints {
-		if isLocalStorage(ep) {
-			anyLocalEp = true
-			break
-		}
-	}
-	if !anyLocalEp {
+	if !isAnyEndpointLocal(endpoints) {
 		fatalIf(errors.New("No endpoint is local to this server"), "No endpoint is local to this server.")
```

Comment:
Same error message.. repeated.  Make them distinct.  One is an actual error and another is a user friendly message.